src/Core/src/System.Diagnostics.CodeAnalysis/DynamicallyAccessedMemberTypes.cs

```diff
@@ -82,6 +82,11 @@ internal enum DynamicallyAccessedMemberTypes
 		/// </summary>
 		NonPublicEvents = 0x1000,
 
+		/// <summary>
+		/// Specifies all interfaces implemented by the type.
+		/// </summary>
+		Interfaces = 0x2000,
```

Comment:
Somehow I had an old copy of this file in https://github.com/dotnet/maui/commit/3c8e158175b1a6efdc3ad8638cec268b85577835, this enum was added in .NET 6.